Specs/blob/blob-storage-2017-07-29.json

```diff
@@ -983,8 +925,8 @@
         "tags": [
           "containers"
         ],
-        "operationId": "Containers_ListBlobs",
-        "description": "The List Blobs operation returns a list of the blobs under the specified container",
+        "operationId": "Containers_ListBlobFlatSegment",
```

Comment:
There should be a Containers_ListBlobHierarchySegment as well. Can we add that as part of this change?  It looks like Containers_ListBlobFlatSegment should not have the delimiter parameter and Containers_ListBlobHierarchySegment should have the delimiter parameter. And then some of the other parameters / request body should only be in one or the other.

Reply:
Yep, I had mentioned that in the description as I was planning to add it in the PR that adds new operations.

Happy to add it here instead.

Reply:
Or we can add a task to this PBI for doing that. I have no preference

Reply:
Okay, in that case nevermind

Reply:
As per face-to-face we will take this in a future PR that adds new APIs.